circle.yml

```diff
@@ -236,7 +236,22 @@ commands:
     steps:
       - run:
           name: "Install codecov"
-          command: sudo pip3 install --break-system-packages --upgrade --quiet --no-cache-dir codecov
+          command: |
+            # TODO: This should go to cpp-build-env image
+            # gpg --no-default-keyring --keyring trustedkeys.gpg --keyserver keyring.debian.org --recv-key ED779869
```

Comment:
I wanted to download the PGP public key by ID but this command is not working on CI (works locally).

Reply:
Done. keyserver.ubuntu.com works.